salt/_states/metalk8s_volumes.py

```diff
@@ -0,0 +1,125 @@
+# coding: utf-8
+'''State module to deal with MetalK8s Volume.'''
+
+import logging
+
+from salt.exceptions import CommandExecutionError
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'metalk8s_volumes'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def present(name, volume):
+    """Ensure that the backing storage exists for the specified volume.
+
+    Args:
+        name   (str): SLS caller name
+        volume (str): Volume name
+
+    Returns:
+        dict: state return value
+    """
+    ret = {'name': name, 'changes': {}, 'result': False, 'comment': ''}
+    # Idempotence.
+    if __salt__['metalk8s_volumes.exists'](volume):
+        ret['result'] = True
+        ret['comment'] = 'storage for volume {} already exists'.format(volume)
```

Comment:
Please capitalize at the beginning of comments

Reply:
My rationale here was that you never now how the log message get mix and matched, and thus having proper sentence (starts with a capital, ends with a period) sometime gives weird message in the end.

But yeah, for Salt that's not really an issue so I'll fix that.

Reply:
Done